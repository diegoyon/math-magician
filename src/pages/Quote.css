.container {
  display: flex;
  flex-direction: column;
  gap: 3rem;
  align-items: center;
  max-width: 80vw;
}

.quote {
  display: flex;
  flex-direction: column;
}

.quote p:nth-child(1) {
  font-size: 2rem;
}

.quote p:nth-child(2) {
  align-self: flex-end;
  padding: 1rem;
  font-style: italic;
}

.quote p:nth-child(2)::before {
  content: "-";
}

.container button {
  padding: 1rem 2rem;
  width: fit-content;
  border: none;
  background-color: brown;
  color: white;
  border-radius: 5px;
  cursor: pointer;
  transition: background-color 1s, color 1s;
}

.container button:hover {
  background-color: white;
  outline: 1px solid brown;
  color: brown;
}
