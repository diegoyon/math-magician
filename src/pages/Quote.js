import React from 'react';

const Quote = () => (
  <>
    <h3>
      Mathematics is not about numbers, equations, computations,
      or algorithms: it is about understanding. -William Paul Thruston
    </h3>
  </>
);

export default Quote;
