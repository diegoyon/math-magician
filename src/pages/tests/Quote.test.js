import renderer from 'react-test-renderer';
import Quote from '../Quote.js';

it('renders correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});