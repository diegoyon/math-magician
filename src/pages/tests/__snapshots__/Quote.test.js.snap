// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quote"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thruston
  </p>
</div>
`;
